stubs/setuptools/setuptools/__init__.pyi

```diff
@@ -54,11 +55,9 @@ class _BuildInfo(TypedDict):
     include_dirs: NotRequired[list[str]]
     cflags: NotRequired[list[str]]
 
-# Pytype fails with the following:
-# find_packages = PackageFinder.find
-# find_namespace_packages = PEP420PackageFinder.find
-def find_packages(where: StrPath = ".", exclude: Iterable[str] = (), include: Iterable[str] = ("*",)) -> list[str]: ...
-def find_namespace_packages(where: StrPath = ".", exclude: Iterable[str] = (), include: Iterable[str] = ("*",)) -> list[str]: ...
```

Comment:
pytype still seems to fail here.

Reply:
Yep :/ I wanted to create or associate an issue on pytype's side if this still fails. So I'll do that before reverting.

Reply:
https://github.com/google/pytype/issues/1840